converters/fromZigbee.js

```diff
@@ -2868,6 +2868,75 @@ const converters = {
             }
         },
     },
+    hgkg_thermostat: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandGetData', 'commandSetDataResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data.dp;
+            const value = tuya.getDataValue(msg.data.datatype, msg.data.data);
+            let temperature;
+            /* See tuyaThermostat above for message structure comment */
+            switch (dp) {
+            case tuya.dataPoints.moesSchedule:
+                return {
+                    program: [
+                        {weekdays_p1: value[0] + 'h:' + value[1] + 'm ' + value[2]/2 + '°C'},
+                        {weekdays_p2: value[3] + 'h:' + value[4] + 'm ' + value[5]/2 + '°C'},
+                        {weekdays_p3: value[6] + 'h:' + value[7] + 'm ' + value[8]/2 + '°C'},
+                        {weekdays_p4: value[9] + 'h:' + value[10] + 'm ' + value[11]/2 + '°C'},
+                        {saturday_p1: value[12] + 'h:' + value[13] + 'm ' + value[14]/2+ '°C'},
+                        {saturday_p2: value[15] + 'h:' + value[16] + 'm ' + value[17]/2 + '°C'},
+                        {saturday_p3: value[18] + 'h:' + value[19] + 'm ' + value[20]/2 + '°C'},
+                        {saturday_p4: value[21] + 'h:' + value[22] + 'm ' + value[23]/2 + '°C'},
+                        {sunday_p1: value[24] + 'h:' + value[25] + 'm ' + value[26]/2 + '°C'},
+                        {sunday_p2: value[27] + 'h:' + value[28] + 'm ' + value[29]/2 + '°C'},
+                        {sunday_p3: value[30] + 'h:' + value[31] + 'm ' + value[32]/2 + '°C'},
+                        {sunday_p4: value[33] + 'h:' + value[34] + 'm ' + value[35]/2 + '°C'},
+                    ],
+                };
+            case tuya.dataPoints.state: // Thermostat on standby = OFF, running = ON
+                return {system_mode: value ? 'cool' : 'off'};
```

Comment:
I've compared this converter with `fz.moes_thermostat`, the only difference I could find was this line.

Replacing `return {system_mode: value ? 'heat' : 'off'};` with below will make this converter obsolete.

```js
if (model.model === 'BAC-002-ALZB') {
  return {system_mode: value ? 'heat' : 'off'};
} else {
  return {system_mode: value ? 'cool' : 'off'};
}
```

Reply:
Thank you.
I've modified fz.moes_thermostat per your instruction and removed fz.hgkh_thermostat